worknenjoy/gitpay
# Feature/payment header s ummary

## Description

> Updated payment component, added a payment header that updates bounty information if bounty exists

## Changes

- two files: task-header and within Card directory, all within client side(frontend)

## Issue

> closes #227

## Impacted Area

> Single Task Dashboard

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create/Import an Issue from gitpay account
- You should be able to see updated bounty from the header section if there is a set bounty, and blank if it does not exist

## Before
> 
![image](https://user-images.githubusercontent.com/20405825/79768371-5e7c5a00-8333-11ea-92a2-a9cbf8b9b25c.png)


## After
> 
![image](https://user-images.githubusercontent.com/20405825/79768302-43a9e580-8333-11ea-9507-b9fe15ab3174.png)
